packages/shared/src/styles/themes.scss

```diff
@@ -72,6 +72,7 @@
         --general-section-3: #{$color-grey-11};
         --general-section-4: #{$color-grey-12};
         --general-section-5: #{$color-grey-2};
+        --general-section-6: #{$COLOR_LIGHT_GRAY_6};
```

Comment:
We already have --general-section-6 .. I have added it recently.

Reply:
it was already there. i didn't add.